bin.js

```diff
@@ -53,6 +55,49 @@ var NOCOMMAND = `
   Run ${clr('bankai --help', 'cyan')} to see all options.
 `.replace(/\n$/, '').replace(/^\n/, '')
 
+var CONFIG_USAGE = `
+
+  ${clr('Configuration', 'bold')}
+
+
+  Bankai is built on top of compilers for scripts, styles and documents. 
+  Each of them can be configured by adding a field to your project's 
+  package.json file.
+
+  These three fields are, respectively: ${clr('"browserify"', 'cyan')}, ${clr('"sheetify"', 'cyan')} and 
+  ${clr('"documentify"', 'cyan')}. Each one should have a configuration object as it's value.
+
+  There is currently one possible configuration field: "transform".
+
+  It can be one of either:
+
+  1. An array of transform names. 
+       ie:  ${clr('[ "babelify" ]', 'cyan')}
+  2. An array of tuples transform name + configuration object.
+       ie: ${clr('[[ "babelify", { "presets": [ "env" ] } ]]', 'cyan')}
+
+
+  Full example:
+
+  ${clr(`{
+    "browserify": {
+      "transform": [
+        [ "babelify", { "presets": [ "env" ] } ]
```

Comment:
I think we should use a different example, since babelify is already included with this config by default. Not sure which transform would be a good alternative tho…

Reply:
Good point! Maybe `browserify-istanbul` ? 
It's hard, all my favorite are already used by Bankai 😂

Reply:
Yeaap lol. Not sure about istanbul since it's used in production builds too. Maybe something like [vueify](https://github.com/vuejs/vueify)?

Reply:
Okay! Made the changes using vueify. The appveyor build failed though, but it seems to be unrelated.